atlas/parts/header.html

```diff
@@ -0,0 +1,27 @@
+<!-- wp:group {"layout":{"type":"constrained"}} -->
+<div class="wp-block-group"><!-- wp:group {"align":"wide","layout":{"type":"flex","flexWrap":"wrap","justifyContent":"space-between"}} -->
+<div class="wp-block-group alignwide"><!-- wp:group {"layout":{"type":"flex","justifyContent":"left","flexWrap":"wrap"}} -->
+<div class="wp-block-group"><!-- wp:site-title {"level":0} /-->
+
+<!-- wp:navigation {"ref":973,"icon":"menu","layout":{"type":"flex","setCascadingProperties":true,"justifyContent":"left"}} /--></div>
+<!-- /wp:group -->
+
+<!-- wp:group {"style":{"layout":{"selfStretch":"fill","flexSize":null}},"layout":{"type":"default"}} -->
+<div class="wp-block-group"><!-- wp:social-links {"iconColor":"contrast","iconColorValue":"#e6e6dc","className":"is-style-logos-only","layout":{"type":"flex","justifyContent":"right","flexWrap":"nowrap"}} -->
```

Comment:
can we use the preset here? I think it's called contrast

Reply:
This is using the preset, its the way the block works :(

Reply:
that is bad!